### PR TITLE
Add Sylius 1.11 support

### PR DESCRIPTION
**Pull request by jakubtobiasz**
This PR adds support for the newest Sylius 1.11.

PR covers such cases like:
- Update composer.json
- Update GitHub workflow
- Add Polyfill for behats

### CODE REVIEW

File: src/DependencyInjection/CompilerPass/SyliusBehatPolyfillCompilerPass.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
```
Comment (senghe) [inline src/DependencyInjection/CompilerPass/SyliusBehatPolyfillCompilerPass.php:3]: One comment is enough 🙃

File: tests/Application/.env [hunk 2]
```diff
@@ -27,3 +27,11 @@ JWT_PASSPHRASE=topsecret
 # Delivery is disabled by default via "null://localhost"
 MAILER_URL=smtp://localhost
 ###< symfony/swiftmailer-bundle ###
+
+###> symfony/messenger ###
+# Choose one of the transports below
+# MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
+#MESSENGER_TRANSPORT_DSN=doctrine://default
+# MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages
+MESSENGER_TRANSPORT_DSN=sync://
```
Comment (senghe) [inline tests/Application/.env:36]: Is it the default option, or uncommented by mistake?

File: tests/Application/config/sylius/1.11/bundles.php [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?php
```
Comment (senghe) [inline tests/Application/config/sylius/1.11/bundles.php:1]: No OS header

**Review by senghe — CHANGES_REQUESTED** [review]

File: tests/Application/.env (hunk 2 above)
Comment (jakubtobiasz) [reply tests/Application/.env:36]: Default option :).

File: tests/Application/config/sylius/1.11/bundles.php (hunk 3 above)
Comment (jakubtobiasz) [reply tests/Application/config/sylius/1.11/bundles.php:1]: None of the files in `config` has OS header. Should I add it to all of them?

**Review by senghe — APPROVED** [review]

**Pull request merged**